pkgs/applications/networking/cluster/nomad/genericModule.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, version
+, sha256
+, vendorSha256
+, nvidiaGpuSupport
+, patchelf
+, nvidia_x11
+}:
+
+buildGoModule rec {
+  pname = "nomad";
+  inherit version;
+  rev = "v${version}";
+
+  subPackages = [ "." ];
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = pname;
+    inherit rev sha256;
```

Comment:
```suggestion

  subPackages = [ "." ];

  src = fetchFromGitHub {
    owner = "hashicorp";
    repo = pname;
    rev = "v${version}";
    inherit sha256;
```

Reply:
Done, but it is rather funny to me, as `inherit rev` came at the suggestion of another committer in the original multi-version PR - making requests for changes like this feel like a rather frustrating task, in absence of a widely agreed upon standard formatting :slightly_smiling_face:

Reply:
Agreed 1000x over